src/test/java/PrefixTreeTest.java

```diff
@@ -0,0 +1,48 @@
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class PrefixTreeTest {
+
+    @Test
+    void addingWordsToTree() throws FileNotFoundException {
+        PrefixTree prefixTree = new PrefixTree();
+        File file = new File("src/test/resources/tests.txt");
+        try (Scanner sc = new Scanner(file)) {
+            while (sc.hasNextLine()) {
+                String currentWord = sc.nextLine();
+                prefixTree.add(currentWord);
+            }
+        }
+        try(Scanner sc = new Scanner(file)) {
+            while(sc.hasNextLine()){
+                String currentWord = sc.nextLine();
+                assertTrue(prefixTree.contains(currentWord));
+            }
+        }
```

Comment:
Here and below: it makes sense to put the contents of the file into a list and then use it from memory. Given that the file is used in both tests, the way to go is to do this before running the tests.